src/components/export/ExportTab/ExportTablesContainer.js

```diff
@@ -0,0 +1,154 @@
+import { Preloader } from "@us-epa-camd/easey-design-system";
+import React, { useEffect, useState } from "react";
+import DataTable from "react-data-table-component";
+import { ArrowDownwardSharp } from "@material-ui/icons";
+
+import { getQATestSummary } from "../../../utils/api/qaCertificationsAPI";
+import { Button } from "@trussworks/react-uswds";
+
+const ExportTablesContainer = ({
+  selectionData,
+  selectedConfig,
+  exportState,
+  setExportState,
+  workspaceSection,
+}) => {
+  const { beginDate: beginDateOption, endDate: endDateOption } = selectionData;
+  const { locations } = selectedConfig;
+  const [qaTestSummaryData, setQATestSummaryData] = useState(
+    exportState.qaTestSummaryRows
+  );
+
+  useEffect(() => {
+    const fetchQATestSummary = async () => {
+      const allPromises = [];
+      locations.forEach((e) => {
+        allPromises.push(
+          getQATestSummary(e.id, beginDateOption, endDateOption)
+        );
+      });
+      Promise.all(allPromises).then((res) => {
+        setQATestSummaryData(res.map((e) => e.data).flat());
+      });
+    };
+    fetchQATestSummary();
+  }, [beginDateOption, endDateOption]);
+
+  const rows = qaTestSummaryData?.map((obj) => {
+    const {
+      id,
+      componentID: componentId,
```

Comment:
You should be using `componentID` , not `componentId`. 

`componentID` stands for 'component_indetifier' and 
`componentId` stands for component record id.